fix epoch/eschaton/takedown dates

binary1230 opened the pull request:
woops, we set these wrong

bds002 committed:
Changed Epoch Dates to match Stock schedule dates

Epoch dates where set for Sun. instead of Friday.

bds002 committed:
Setting takedown to Eschaton

Per conversation in Slack #Devops,

Making Uber_takedown = eschaton

binary1230 commented on event-magstock.yaml:
```diff
@@ -23,16 +23,16 @@ uber::config::year: 6
 #uber::config::printed_badge_deadline: ''
 #uber::config::dealer_reg_start: ''
-uber::config::uber_takedown: '2016-06-17'
+uber::config::uber_takedown: '2016-06-19 18'
```
Not sure we should change takedown.  A bunch of stuff assumes that uber_takedown is before the start of the event, like:

```
StopsEmail('Your {EVENT_NAME} shift schedule', 'shifts/schedule.html',
           lambda a: c.SHIFTS_CREATED and days_before(1, c.UBER_TAKEDOWN) and a.weighted_hours)
```

If we changed to the 19th, shift schedules won't be emailed out til the Saturday of the event.

If this is to fix the transfer of badges, we should probably just make the badge transfer deadline be a separate new date in this list so it can be set independently of the start of the event.

kitsuta replied:
OK, having finally had a look at the Prime settings, we do indeed have uber_takedown set to the day before the event. So let's do that here and worry about the other stuff separately.

kitsuta committed:
Make uber_takedown proportional to Prime's

Looking at https://github.com/magfest/production-config/blob/master/event-prime.yaml#L50, the uber_takedown date actually is set to the day before the epoch, so we'll do the same here.

Keep in mind, uber_takedown is a text configuration setting, and nothing will actually shut down.

kitsuta commented:
+1

EliAndrewC commented:
+1 (belatedly)
